test/correctness/cross_compilation.cpp

```diff
@@ -37,7 +37,7 @@ int main(int argc, char **argv) {
 
         #ifndef _MSC_VER
         std::string object_name = "test_object_" + t + ".o";
```

Comment:
Yeah, that code is a little too clever. Fixed in trunk and merged into the
branch.

On Thu, Jan 21, 2016 at 2:09 PM, Zalman Stern <notifications@github.com>
wrote:

> In test/correctness/cross_compilation.cpp
> <https://github.com/halide/Halide/pull/1002#discussion_r50472457>:
>
> > @@ -37,7 +37,7 @@ int main(int argc, char **argv) {
> >
> >          #ifndef _MSC_VER
> >          std::string object_name = "test_object_" + t + ".o";
>
> Can we fix this to not form the string ".obj" via concatenation? It is
> really nice to be able to grep code for "obj" and find all occurrences and
> in this case it seems trivial to just append ".o" or ".obj" based on the
> platform. (Yes, I am aware that the problem is not introduced by this PR,
> but it seems easy to fix here.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/halide/Halide/pull/1002/files#r50472457>.
>